ormconfig.js

```diff
@@ -1 +1,8 @@
-export default (await import('./dist/lib/database/database.config.js')).AppDataConfig;
+import { AppDataConfig } from '#lib/database/database.config';
```

Comment:
This should probably be `import()`'d after `setup()` so the environment variables are loaded